documentation/src/main/asciidoc/topics/ref_rest_xsite.adoc

```diff
@@ -1,14 +1,13 @@
 [id='rest_v2_xsite_sites_status']
```

Comment:
@gustavonalle heads up on doc changes to rest xsite. this includes the changes for getting state transfer status. please let me know if any changes are needed. cheers.